src/components/result/LabelResult.tsx

```diff
@@ -41,12 +41,17 @@ const LabelResult = (props: PropsType): JSX.Element => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [score]);
 
+    const rescale = (valueStart: number, domainStart: number[], domainEnd: number[]): number => {
```

Comment:
- Definitely not a great name as this could also mean scaling the image/svg, perhaps `calcScaledValue`?
- `valueStart` can become `value`

Reply:
Well, the official name is Linear Interpolation, which is lerp in short. But this may be confusing for those who are not used to this math. Maybe rescaleNumber? 'calc' is a very generic term, so I would only use it when it is really useful.